ui/component/fileActions/view.jsx

```diff
@@ -100,84 +99,124 @@ function FileActions(props: Props) {
       push(`/$/${PAGES.REPOST_NEW}?from=${encodeURIComponent(uri)}&redirect=${pathname}`);
     }
   }
 
-  const lhsSection = (
-    <>
-      {ENABLE_FILE_REACTIONS && <FileReactions uri={uri} livestream={isLivestreamClaim} />}
-      <ClaimSupportButton uri={uri} fileAction />
-      <ClaimCollectionAddButton uri={uri} fileAction />
-      {!hideRepost && (
+  const repostButton = () =>
+    isMobile ? (
+      <MenuItem className="comment__menu-option" onSelect={handleRepostClick}>
+        <div className="menu__link">
+          <Icon aria-hidden icon={ICONS.REPOST} />
+          {claim.meta.reposted > 1 ? __(`%repost_total% Reposts`, { repost_total: claim.meta.reposted }) : __('Repost')}
+        </div>
+      </MenuItem>
+    ) : (
+      <Tooltip title={__('Repost')} arrow={false}>
         <Button
           button="alt"
           className="button--file-action"
           icon={ICONS.REPOST}
           label={
-            claim.meta.reposted > 1 ? __(`%repost_total% Reposts`, { repost_total: claim.meta.reposted }) : __('Repost')
+            claimMeta.reposted > 1 ? __(`%repost_total% Reposts`, { repost_total: claimMeta.reposted }) : __('Repost')
           }
-          description={__('Repost')}
-          requiresAuth={IS_WEB}
+          requiresAuth
           onClick={handleRepostClick}
         />
-      )}
-      <Button
-        className="button--file-action"
-        icon={ICONS.SHARE}
-        label={isMobile ? undefined : __('Share')}
-        title={__('Share')}
-        onClick={() => openModal(MODALS.SOCIAL_SHARE, { uri, webShareable, collectionId })}
-      />
-    </>
-  );
+      </Tooltip>
+    );
 
-  const rhsSection = (
-    <>
-      {/* @if TARGET='app' */}
-      <FileDownloadLink uri={uri} />
-      {/* @endif */}
-      {claimIsMine && (
-        <Button
-          className="button--file-action"
-          icon={ICONS.EDIT}
-          label={isLivestreamClaim ? __('Update or Publish Replay') : __('Edit')}
-          navigate={`/$/${PAGES.UPLOAD}`}
-          onClick={() => {
-            prepareEdit(claim, editUri, fileInfo);
-          }}
-        />
-      )}
-      {showDelete && (
+  const myClaimButtons = () =>
+    isMobile ? (
+      <>
+        <MenuItem className="comment__menu-option" onSelect={() => doEditForChannel(claim, editUri)}>
+          <div className="menu__link">
+            <Icon aria-hidden icon={ICONS.EDIT} />
+            {isLivestreamClaim ? __('Update or Publish Replay') : __('Edit')}
+          </div>
+        </MenuItem>
+
+        <MenuItem className="comment__menu-option" onSelect={() => doOpenModal(MODALS.CONFIRM_FILE_REMOVE, { uri })}>
+          <div className="menu__link">
+            <Icon aria-hidden icon={ICONS.DELETE} />
+            {__('Delete')}
+          </div>
+        </MenuItem>
+      </>
+    ) : (
+      <>
+        <Tooltip title={isLivestreamClaim ? __('Update or Publish Replay') : __('Edit')} arrow={false}>
+          <div style={{ margin: '0px' }}>
```

Comment:
We can create a class for this "zero margin" div, even if it is trivial.  Let's not break the pattern by defining the styles in the component itself.  It'll also be easier to adjust if we need to tweak it in the future (just edit the css instead of finding all instances of margin=0px).

_Component reflects prop state --> chooses what css class to use for those states --> we define the styles in the css files._

Reply:
Things might change if we move to Tailwind or some other framework, but for now it's best to maintain the pattern.